README.md

```diff
@@ -35,10 +48,50 @@ collections:
 
 **NOTE**: For Ansible 2.9, you may not see deprecation warnings when you run your playbooks with this collection. Use this documentation to track when a module is deprecated.
 
+### Using ``ansible.network`` meta collection to install supported network content collections.
```

Comment:
Versions aren't pinned so might want to add a note that the latest available will be installed

Reply:
Updated!